.pre-commit-config.yaml

```diff
@@ -1,63 +1,65 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0  # Use the ref you want to point at
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0 # Use the ref you want to point at
     hooks:
-    -   id: check-added-large-files
-    -   id: check-case-conflict
-    -   id: check-executables-have-shebangs
-    -   id: check-merge-conflict
-    -   id: check-toml
-    -   id: check-vcs-permalinks
-    -   id: check-yaml
-    -   id: detect-private-key
-    -   id: end-of-file-fixer
-    -   id: fix-byte-order-marker
-    -   id: mixed-line-ending
-    # -   id: trailing-whitespace
-    # Python-specific
-    - id: check-ast
-    - id: check-docstring-first
-    - id: debug-statements
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-vcs-permalinks
+      - id: check-yaml
+      - id: detect-private-key
+      # - id: end-of-file-fixer
```

Comment:
I commented out eof fixer as it conflicts with the string comparison checks we do in `.err` files, similar to trailing whitespace below.